test/integration/extensibleEnums.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { expect } from "chai";
+import { ExtensibleEnumsClient } from "./generated/extensibleEnums/src";
+
+describe("Integration tests for extensible enums", () => {
+  let client: ExtensibleEnumsClient;
+
+  beforeEach(() => {
+    client = new ExtensibleEnumsClient();
+  });
+
+  it("sends an unexpected enum value successfully", async () => {
+    const response = await client.pet.getByPetId("casper");
+    expect(response).to.deep.equal({

```

Comment:
Would you mind switching to assert? This is in order to be consistent with the azure-sdks-for-js repo